decidim-system/app/forms/decidim/system/register_organization_form.rb

```diff
@@ -18,11 +20,29 @@ class RegisterOrganizationForm < UpdateOrganizationForm
       attribute :users_registration_mode, String
       attribute :force_users_to_authenticate_before_access_organization, Boolean
 
-      validates :organization_admin_email, :organization_admin_name, :name, :host, :reference_prefix, :users_registration_mode, presence: true
+      validates :organization_admin_email, :organization_admin_name, :name, :reference_prefix, presence: true
+      validates :name, presence: true
```

Comment:
In the register organization we have this line, and in UpdateOrganizationForm, we are using `validate :validate_organization_name_presence` ( the method you ask me to remove )